tests/serverpod_test_server/test_integration/test_tools/session_copy_with_test.dart

```diff
@@ -19,15 +21,23 @@ void main() {
             () async {
           await expectLater(
             modifiedSessionBuilder.build().authenticated,
-            completion((a) => a.userId == 123),
```

Comment:
Since the `completion` callback was not typed here (`dynamic`), it did not show that the import for the extension was missing. And without a type, the linter tool would remove the extension import, as it did not know it was used here. But I think in practice an `AuthenticationInfo` passed as `dynamic`  will not be thing to worry about.

Reply:
I kept the typed `isA` here, as loosing the types in `completion` seems suboptimal. Luckily such things will be better with https://pub.dev/packages/checks in the future :)